internal/compiler-bridge/src/main/scala/xsbt/DelegatingReporter.scala

```diff
@@ -51,20 +52,24 @@ private object DelegatingReporter {
     override val startColumn = o2oi(startColumn0)
     override val endLine = o2oi(endLine0)
     override val endColumn = o2oi(endColumn0)
-    override def toString =
-      (sourcePath0, line0) match {
-        case (Some(s), Some(l)) => s + ":" + l
-        case (Some(s), _)       => s + ":"
-        case _                  => ""
-      }
+
+    override def toString = (sourcePath0, line0) match {
+      case (Some(s), Some(l)) => s + ":" + l
+      case (Some(s), _)       => s + ":"
+      case _                  => ""
+    }
```

Comment:
I prefer the previous style here too.

Reply:
This is `newlines.beforeMultilineDef`.  I've gone with `fold`, but there's also `keep`, which I think is documented as "may be buggy", `classic`, which I assume will be removed eventually?, and `unfold` which would suck for many `= this match {` / `= foo match {` definitions.

So we can either take a risk by switching to `keep`, take another risk going with `classic`, or stick with `fold`.

Personally if a definition is fully defined in terms of a match (and that initial RHS fits on the RHS of `=`) then I think this looks neat.